Formula/vulkan-loader.rb

```diff
@@ -0,0 +1,35 @@
+class VulkanLoader < Formula
+  desc "Vulkan ICD Loader"
+  homepage "https://github.com/KhronosGroup/Vulkan-Loader"
+  url "https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.220.tar.gz"
+  sha256 "fe5b65e5d88febb1220ae59de363ed8b2794f8c861d6d74efb14aecefd464559"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Loader.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.10" => :build
+  depends_on "vulkan-headers"
+
+  def install
+    system "cmake", ".", "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+           "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
+            *std_cmake_args
+    system "make", "install"
```

Comment:
Let's follow the pattern used by `brew create --cmake`.
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
                    "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
                    *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--build", "install"
```